_data/navigation.yml

```diff
@@ -81,6 +81,11 @@ wiki:
         url: /wiki/actuation/uln2003a-motor-controller/
       - title: Vedder Electronic Speed Controller
         url: /wiki/actuation/vedder-electronic-speed-controller/
+  - title: Deep Learning
+    url: /wiki/deep-learning/custom-semantic-data
```

Comment:
Adding a category makes sense. However, we should use *Machine Learning* in place of *Deep Learning.*